pkgs/by-name/ne/neonmodem/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+}:
+
+buildGoModule rec {
+  pname = "neonmodem";
+  version = "1.0.6";
+
+  src = fetchFromGitHub {
+    owner = "mrusme";
+    repo = "neonmodem";
+    tag = "v${version}";
+    hash = "sha256-VLR6eicffA0IXVwEZMvgpm1kVmrLYVZOtq7MSy+vIw8=";
+  };
+
+  vendorHash = "sha256-pESNARoUgfg5/cTlTvKF3i7dTMIu0gRG/oV4Ov6h2cY=";
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  postInstall = ''
```

https://github.com/NixOS/nixpkgs/issues/308283
```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

---

Whoops, hadn't even thought about that. Thanks